### ISSUE TITLE
Illegal instruction at address 0x7ff6bed2d1a5

### ISSUE THREAD

**Issue by demkom58** [oven-sh/bun#9594]
### What version of Bun is running?

Latest at this moment

### What platform is your computer?

Microsoft Windows NT 10.0.19045.0 x64

### What steps can reproduce the bug?

1. irm bun.sh/install.ps1 | iex
2. 
```
################################################################################################################# 100.0%
Illegal instruction at address 0x7ff6bea42b32
Illegal instruction at address 0x7ff6bed2d1a5
Illegal instruction at address 0x7ff6bed2d1a5
Illegal instruction at address 0x7ff6bed2d1a5
Illegal instruction at address 0x7ff6bed2d1a5
Illegal instruction at address 0x7ff6bed2d1a5
Illegal instruction at address 0x7ff6bed2d1a5
Illegal instruction at address 0x7ff6bed2d1a5
---- 4290 lines skipped ----
Illegal instruction at address 0x7ff6bed2d1a5
Illegal instruction at address 0x7ff6bed2d1a5
Stack Overflow
Illegal instruction at address 0x7ff6bed2d1a5
Install Failed - could not verify bun.exe
The command 'C:\Users\demko\.bun\bin\bun.exe --revision' exited with code -1073741819
```

Even simple launch of bun cause same error.

### Additional information

It is only the guess, but it maybe due to missing SIMD instructions (if they were used). 
Here is [spec](https://www.techpowerup.com/cpu-specs/phenom-ii-x4-965-be-125w.c676) of my processor.

**Comment by Eboubaker** [comment]
it happened to me after I changed my processor (I have moved my drive from my laptop to my desktop pc)

**Comment by demkom58** [comment]
> it happened to me after I changed my processor (I have moved my drive from my laptop to my desktop pc)

Can you share link on spec of your processor? You can find it [here](https://www.techpowerup.com/cpu-specs/).

**Comment by rodnoycry** [comment]
Tried to run ```bun i``` in my project that I previously made in WSL, and got this output with error ```Illegal instruction at address 0x7ff6b30c84d1```:
```
bun install v1.1.0 (5903a614)
  🚚 natural-compare... Illegal instruction at address 0x7ff6b30c84d1
  🚚 resolve-from... ???:?:?: 0x7ff6b30c876f in ??? (bun.exe)
???:?:?: 0x7ff6b25e465e in ??? (bun.exe)
???:?:?: 0x7ff6b27877e6 in ??? (bun.exe)
???:?:?: 0x7ff6b257d834 in ??? (bun.exe)
???:?:?: 0x7ff6b2736824 in ??? (bun.exe)
  🚚 lines-and-columns... ???:?:?: 0x7ffe6cfa7343 in ??? (KERNEL32.DLL)
???:?:?: 0x7ffe6e5026b0 in ??? (ntdll.dll)
Illegal instruction at address 0x7ff6b30c84d1
  🚚 styled-jsx... ???:?:?: 0x7ff6b30c876f in ??? (bun.exe)
???:?:?: 0x7ff6b25e465e in ??? (bun.exe)
???:?:?: 0x7ff6b27877e6 in ??? (bun.exe)
???:?:?: 0x7ff6b257d834 in ??? (bun.exe)
???:?:?: 0x7ff6b2736824 in ??? (bun.exe)
```
OS version: Windows 10 Pro Version 22H2 (OS Build 19045.4170)
CPU: https://www.techpowerup.com/cpu-specs/celeron-j4125.c3280

**Comment by demkom58** [comment]
I tested it in WSL, so this is not Windows specific problem. I suppose windows label can be removed.

```
Welcome to Ubuntu 22.04.3 LTS (GNU/Linux 5.15.146.1-microsoft-standard-WSL2 x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage


invokegs@locust:~$ curl -fsSL https://bun.sh/install | bash
######################################################################## 100.0%
bun was installed successfully to ~/.bun/bin/bun
bash: line 159:   529 Illegal instruction     IS_BUN_AUTO_UPDATE=true $exe completions &> /dev/null
Run 'bun --help' to get started
invokegs@locust:~$ bun --help
Illegal instruction
```

**Comment by Jarred-Sumner** [comment]
> Nov 7th, 2009

It is indeed due to lack of CPU instructions with SIMD support

You can give the `-baseline` build of Bun a try, but most likely you were already using that because the install script attempts to automatically detect it

We do not currently support CPUs which lack SSE 4.2/AVX support and are unlikely to

**Comment by Eboubaker** [comment]
> We do not currently support CPUs which lack SSE 4.2/AVX support and are unlikely to

I don't think it only happens because of SSE, as I mentioned I fist installed bun on my laptop it worked fine. then when I moved my drive to my desktop PC it made that error so I removed bun and installed it again now it works fine. I don't know why the reinstall  was necessary.
My laptop CPU is I5-4210U My Desktop is i7-3770K

**Comment by Eboubaker** [comment]
> Can you share link on spec of your processor? You can find it [here](https://www.techpowerup.com/cpu-specs/).
My Desktop: https://www.techpowerup.com/cpu-specs/core-i7-3770k.c1001
My Laptop: https://www.techpowerup.com/cpu-specs/core-i5-4210u.c2028

The website reports they both support SSE4.2

**Comment by robobun** [comment]
This is the same problem tracked in #7179: on an x86_64 CPU without SSE4.2 (AMD Phenom II X4 965 here, AMD A6-3410MX and Phenom II X6 there) every Bun invocation, including the baseline build, dies with `Illegal instruction` right after the install script runs, and as noted in the comments the identical failure reproduces with the Linux build under WSL. The missing-SSE4.2 crash on startup (graceful-exit fix tracked in #14745) will continue to be tracked in #7179.

Duplicate of #7179

Please add a 👍 or any extra details over there so it all stays in one place. If this is actually a different problem, say so here and we'll reopen.
